lib/vyasa_web/components/source_content/verses.ex

```diff
@@ -96,14 +96,13 @@ defmodule VyasaWeb.Content.Verses do
               <%= Struct.get_in(Map.get(elem, :node, @verse), elem.field) %>
             </dd>
             <div
-              :if={@verse.binding}
+              :if={@verse.binding && (@verse.binding.node_id == Map.get(elem, :node, @verse).id &&
```

Comment:
looking at the thickness, this it make sense to put this as an assigned bool?

Reply:
will be merging regardless

Reply:
yeah probably scope it out as a function that we can call in to guard the ui